ddd-kata.md

```diff
@@ -0,0 +1,197 @@
+7. There are three types of terminals:
+   1. The terminals at the entrances have a way for the customer to communicate the vehicle details to the system.
+   2. Customer drive into the parking garage / lot via entrance terminals where the collect the tickets
+   3. The terminals near the exits have a way to pay the ticket.
+   4. Detail out the actual payment process like select payment method, put in credit card, enter PIN are not relevant here.
+8. The terminals at the exit gates collect the paid ticket.
+9. The ticket has a read/write magnetic stripe containing information including a position to park.
+10. The magnetic stripe can be changed by the terminals.
+11. Parking garages have a parking guiding system, that guides customers to the correct parking spot. The guiding system recognizes the vehicle and guides it to the right parking spot.
+12. Parking lots do not have such a system.
+
+As domain expert, please stay from requirements perspective within these guardrails. You can detail them out as much as you want.
+
+### The Process
+
+#### Define the Team
+
+![Team Setup](./images/team-setup.png)
+
+As a **Moderator / Facilitator**, your task is to organize the workshops and conduct the workshops.
+
+As a **Domain Expert**, your tasks are:
+
+* Mimics the expert for parking lot software & management
+* Takes business decisions, has the last say on requirements
+* Researches and creates requirements
+
+As a **Team Member** your tasks are:
+
+* Interview the Domain Expert
+* Create the deliverables
+* Participate actively in all workshops
+
+#### Plan your timeline
+
+![Timeline Proposal](./images/timeline-proposal.png)
+
+* Plan to have ~4h a week of modelling sessions
+* Work iteratively (e.g. while working on Aggregate Canvas you might want to adjust your Event Storming)
+* As a moderator, strife for consensus and alignment in the team
+* Plan your time carefully. Timebox your efforts.
+* Most importantly, have fun modelling.
+
+#### How to run an Event Storming Workshop
+
+Event storming is a collaborative workshop method that helps to explore complex business domains and model them using events. An event is an action that occurred in the system at a specific time. Event storming can be conducted in two phases: a high-level event storm to identify the domains and then a detailed event storm to focus on a core domain.
+
+The high-level event storm consists of these steps:
+
+1. **Chaotic Exploration**: Invite the right people from different perspectives and roles, such as domain experts, developers, testers, etc. Provide unlimited modeling space, such as a wall or a whiteboard. Ask everyone to write down relevant events in the domain and the process that they are analyzing on sticky notes. Use different colors for different types of events, such as past, present, future, etc. Encourage everyone to participate and share their ideas without filtering or criticizing.
```

Comment:
> Use different colors for different types of events, such as past, present, future, etc.

I'd challenge this recommendation. Since we kinda make people transition from high-level to low-level with the same event spine, introducing multiple colors can lead to confusion (at the latest once they introduce commands and policies).

Reply:
You are right